src/test/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployerTests.java

```diff
@@ -37,7 +37,7 @@
  *
  * @author Moritz Schulze
  */
-public class KubernetesDeployerTests {
+public class AbstractKubernetesDeployerTests {
 
```

Comment:
Tests with name AbstractXxx aren't included during regular builds, renaming to RunAbstractKubernetesDeployerTests on merge